gratipay/renderers/jinja2_text.py

```diff
@@ -0,0 +1,5 @@
+import aspen_jinja2_renderer as base
+
+# We're actually fine with the default behavior for text/plain.
+Renderer = base.Renderer
+Factory = base.Factory
```

Comment:
Dropped in https://github.com/gratipay/gratipay.com/commit/7b250f6bb8a5f7d73cc708493c7024ba40ccbe42.